src/lib/service/generateApexTransport.js

```diff
@@ -68,13 +68,13 @@ function generateClasses(config) {
 
 	const
 		avscFilesPath = path.resolve(process.cwd(), config.argv.inputUrl),
-		files = getAvscFilesOnPathRecursively(avscFilesPath),
+		files = read.readFilesWithExtension(avscFilesPath, '.avsc'),
 		parsedSchemas = parseSchemas(files),
-		result = generate(parsedSchemas),
+		result = generate.generate(parsedSchemas),
 		outputPath = path.resolve(process.cwd(), config.argv.outputUrl);
 
-	return overwriteFile(outputPath, 'OrizuruTransport.cls', result.cls)
-		.then(() => overwriteFile(outputPath, 'OrizuruTransport.cls-meta.xml', result.xml))
+	return overwriteFile.overwriteFile(outputPath, 'OrizuruTransport.cls', result.cls)
```

Comment:
I understand the desire to export more than the single function, but the naming here looks a bit weird...

Reply:
Agreed. I've aliased overwriteFile so that this doesn't look quite as strange.